tests/ui/modules/module-use-nested-groups.rs

```diff
@@ -15,10 +17,13 @@ mod a {
 }
 
 // Test every possible part of the syntax
-use a::{B, d::{self, *, g::H}};
-
 // Test a more common use case
-use std::sync::{Arc, atomic::{AtomicBool, Ordering}};
```

Comment:
I think this test is intended to test the nested syntax, so these should probably stay as-is with `#[rustfmt::skip]`. See its addition in 91ba8b42fcce0491fa8a1b95e4088bcabf9abf4a.